### DISCUSSION TITLE
Unable to setup Graphql API for runtime using SQLite

### DISCUSSION THREAD

**Discussion by ancashoria** [keystonejs/keystone#7439] [Questions]
Hi,
I'm using Keystone in embedded mode with SQLite and I'm trying to use the Graphql api at runtime like it's described here, but it doesn't work for me.
https://keystonejs.com/docs/walkthroughs/embedded-mode-with-sqlite-nextjs#bonus-add-the-graph-ql-api-to-the-frontend

I'm getting this error when accessing `/api/graphql` endpoint
<img width="940" alt="Screen Shot 2022-04-06 at 4 40 13 PM" src="https://user-images.githubusercontent.com/4144110/161988079-b3b5ee23-ecc1-47b5-8dde-5a4eaeeb5692.png">

Can anyone help?

**Comment by emmatown** [comment]
Could you provide a repository with your code that causes this error?
